### PR TITLE
Pivot table conditional formatting

### LINKED ISSUE #15693: Pivot Table with Conditional Formatting
In table visualization we have **Conditional Formatting** option:

image

But if if I choose **Pivot table** this options is not available:

image

image

It would be great to have **Conditional Formatting** in **Pivot Table** because normal tables don't show the **Total** in footer. And I need Total and Formatting at the same time!

Thank you!

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

### PR DESCRIPTION
**Pull request by alxnddr**
[Mockups](https://www.figma.com/file/leZMwxcyiFCT6UGaSby4Ju/Bring-select-chart-types-closer-to-feature-parity?node-id=101%3A19643)
Epic: https://github.com/metabase/metabase/issues/24877
Closes https://github.com/metabase/metabase/issues/15693

## Changes

- Add conditional formatting to pivot tables
  - "Highlight the whole row" does not apply to pivot tables since it makes less sense and it does not work for regular pivoted tables
- Update the look of pivot tables

## How to verify

- New -> Question -> Orders
- Summarize by: Created at, Total, User -> Source
- Add metrics: Count, Average of subtotal
- Select the pivot table visualization type
- Apply conditional formatting, ensure it works as defined
- Try other configurations of pivot tables to check the styling
- Add the pivot table on a dashboard, enter full-screen mode, enable night mode

#### Conditional formatting

Screen Shot 2022-08-25 at 3 40 13 PM

Screen Shot 2022-08-25 at 2 51 35 PM

#### Dark mode
Screen Shot 2022-08-25 at 2 25 33 PM

### CODE REVIEW

File: frontend/src/metabase/visualizations/visualizations/PivotTable.styled.tsx [hunk 1]
```diff
@@ -1,5 +1,11 @@
+import { css } from "@emotion/react";
 import styled from "@emotion/styled";
-import { color } from "metabase/lib/colors";
+import { color, alpha } from "metabase/lib/colors";
+import { Collection } from "react-virtualized";
+
+const EMPHASIZED_BG_COLOR = "#F8FAFB";
```
Comment (ranquild) [inline frontend/src/metabase/visualizations/visualizations/PivotTable.styled.tsx:6]: Can this color be computed from existing ones?

**Review by ranquild — CHANGES_REQUESTED** [review]
@alxnddr I've tried using new formatting settings with a slightly different query and it doesn't seem to work for me. The result below is unexpected, is it?

Screenshot 2022-08-25 at 15 57 32

I also see some strange columns
Screenshot 2022-08-25 at 15 59 18

**Comment by alxnddr** [comment]
@ranquild based on your screenshots, you don't have the latest changes since the pivot table has old styling

**Review by ranquild — APPROVED** [review]

**Comment by mvtemple** [comment]
hi @alxnddr - I don't see this in my UI yet, has this been released?

**Comment by alxnddr** [comment]
Hey @mvtemple, this feature will be included into the upcoming x.45.0 release

**Pull request merged**